python/lsst/obs/base/makeRawVisitInfoViaObsInfo.py

```diff
@@ -78,16 +78,14 @@ def __init__(self, log=None, doStripHeader=False):
         self.log = log
         self.doStripHeader = doStripHeader
 
-    def __call__(self, md, exposureId=None):
+    def __call__(self, md):
         """Construct a VisitInfo and strip associated data from the metadata.
 
         Parameters
         ----------
         md : `lsst.daf.base.PropertyList` or `lsst.daf.base.PropertySet`
             Metadata to pull from.
             May be modified if ``stripHeader`` is ``True``.
-        exposureId : `int`, optional
-            Ignored.  Here for compatibility with `MakeRawVisitInfo`.
```

Comment:
Is the (API?) compatibility still going to be a problem?

Reply:
`MakeRawVisitInfo` no longer exists, so I don't think so.